<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatientWindow</class>
 <widget class="QDialog" name="AddPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert new patient</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>348</width>
     <height>339</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSurname">
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtSurname"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSex">
        <property name="text">
         <string>Sex</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rbtnMale">
          <property name="text">
           <string>Male</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnFemale">
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblBirthday">
        <property name="text">
         <string>Date of birth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateBirthday"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblBirthplace">
        <property name="text">
         <string>Place of birth</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmbBirthplace"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblFiscalCode">
        <property name="text">
         <string>Fiscal Code</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtFiscalCode"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblResidence">
        <property name="text">
         <string>Residence</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtResidence"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblEmail">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="txtEmail"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblPhone">
        <property name="text">
         <string>Phone number</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="txtPhone"/>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnInsert">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
